README.rst

```diff
@@ -0,0 +1,56 @@
+Copr Builder
+============
+
+A simple program for building projects from Git repository in Copr.
+
+Usage
+-----
+
+::
+
+  usage: copr-builder.py [-h] [-v] [-p [PROJECTS [PROJECTS ...]]] [-c CONFIG]
+
+  Copr builder
+
+  optional arguments:
+    -h, --help            show this help message and exit
+    -v, --verbose         print debug messages
+    -p [PROJECTS [PROJECTS ...]], --projects [PROJECTS [PROJECTS ...]]
+                          projects to build; if not given, all projects from
+                          config will be built
+    -c CONFIG, --config CONFIG
+                          config file location
+
+Config file structure
+--------------------
+
+::
+
+  [project_name]
+  copr_user =
+  copr_repo =
+  package =
+  git_url =
+  archive_cmd =
+  git_branch =
+  git_merge_branch =
+
+  [another_project_name]
+  copr_user =
```

Comment:
I think it would be nice to have here example values, like `https://github.com/user/example-project` and similar.